.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 4,
```

Comment:
I think this was my bad but I like 2 for tabWidth. 4 feels to spaced out.

Reply:
Changed tabWidth to 2. Good call - it looked off for sure. I went ahead and saved the tabWidth changes in /queries/api.js as well.